src/Autofac.Extras.MvvmCross/Autofac.Extras.MvvmCross.csproj

```diff
@@ -0,0 +1,62 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Autofac extension supporting MvvmCross dependency injection.</Description>
+    <VersionPrefix>4.0.2</VersionPrefix>
```

Comment:
There are breaking changes here so might need to go for `5.0.0`.

Reply:
Good point, I'll update this to `5.0.0`.